### DISCUSSION TITLE
(Feature Request) New enemy: the Captivmind

### DISCUSSION THREAD

**Discussion by inferknob** [FakeFishGames/Barotrauma#12573] [Ideas and suggestions]
first of all that name is very much so optional if you can think of literally anything better please change it

The Captivmind is a multi-part organism that refers to 3 parts of the creature, I will go through all of them in detail.

**The Nucleus**
The Nucleus is basically the brain of everything. Once killed, every other piece will die, much like a thalamus. It has an armored shell made of wrecked submarine parts and will need to be damaged either by heavy ammo (railgun or other), explosives, or damaging a socket where a limb used to be. Looking at a living nucleus will give you a mild case of Psychosis and Captivated (I will explain captivated later, its a related debuff)

**Drones**
Drones are, well, drones. Not metallic ones by rather mobile entities that will infect your ships hull. They will try to get in undetected by breaking in through leaking hulls and have the surprising ability to repair them with a biological adhesive. They will just crawl towards people. Being near one for too long will give you Psychosis and Captivated, and they like to attack and control engines. I should explain what these are for but first I need to explain captivated:

**Captivated**
If you have captivated, you become compelled to draw closer to the thing mesmerizing you or if your in a sub you will try to pilot it towards the nucleus. This will auto control you but you can resist it by trying to move around erratically. For example: by spamming A and D and pressing spacebar and whatnot.

**Drones: Part 2: The Dronening**
So really their job is to bring you to the nucleus. However: if this fails and people in the sub are not able to captivated: they will destroy the engine and ballast pumps, and pilot the sub themselves to the nucleus. Then their job is over.

**Appendages**
We got 2 types of appendages here, which I was too bad at naming things to name: The Hull Breacher and the People Taker. Name them yourselves, I genuinely cannot. Appendages are connected to the nucleus, there is 2 of each and have medium health. Also, when they are destroyed, the socket they where originally put on in the nucleus will be damageable and will do 3x damage to the nucleus than regularly. Anyways:

**Hull breacher**
Does everything you'd think it'd do. It smashes and destroys the hull of the ship and forcibly tries to flood the whole ship.

**People Taker**
also does exactly what you thought it did. Steals people, and tries to crush and then eat their body.

Okay, that's pretty much it. If you kill a captivmind then the nucleus will die and drop some supplies and metal, for it has a wrecked ship hull armor on it. I'm thinking there'd be an achievement, maybe, "Broke free" "Kill a Captivmind".

Pretty much what it'd look like. Rough sketch on canvas.

![canvas (2)](https://github.com/Regalis11/Barotrauma/assets/137335926/f8308da0-b3d5-4ae7-bbc7-d60911e4ef1c)

**Comment by Hazmat616** [comment]
Sounds a lot like the legacy carrier

**Reply by inferknob** [reply]
What... What's the carrier?

**Reply by inferknob** [reply]
That thalamus thing? How is this like a thalamus?

**Reply by Hazmat616** [reply]
> What... What's the carrier?

https://barotraumagame.com/wiki/Carrier_(Legacy)
